### PR TITLE
Extending tables with sent data for each type ofSCC  PROXY (bsc#1214469)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Are there any relevant issues/feature requests?

* bsc#1214469

### CODE REVIEW

File: scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml [hunk 1]
```diff
@@ -148,15 +148,56 @@
...
+      The following example shows a JSON representaion of hypervisor data that
+      is being collected.
+    </para>
+
+<screen>
+{
+    "virtualization_hosts": [
+        {
+            "identifier": "SCC_06bc59754...",
+            "group_name": "suma-02.mgr.example.com",
```
Comment (mcalmer) [inline scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml:162]:
```suggestion
            "group_name": "group.example.com",
```

File: scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml [hunk 2]
```diff
@@ -148,15 +148,56 @@
...
+            "properties": {
+                "name": "suma-02-a.mgrexample.com",
```
Comment (mcalmer) [inline scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml:164]:
```suggestion
                "name": "host.example.com",
```

File: scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml [hunk 3]
```diff
@@ -148,15 +148,56 @@
...
+                "threads": 128,
+                "ram_mb": 515233,
+                "type": "KVM"
+            },
+            "systems": [
+                {
+                    "uuid": "47fc43cc-c704-4e08-82ee-0e1694XXXYYZ",
+                    "properties": {
+                        "vm_state": "running",
+                        "vm_name": "sumadocker5"
```
Comment (mcalmer) [inline scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml:177]:
```suggestion
                        "vm_name": "vm1"
```

File: scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml [hunk 4]
```diff
@@ -148,15 +148,56 @@
...
+                    }
+                },
+                {
+                    "uuid": "a914d996-fb27-4d74-8ba7-fde99eXXYYZZ",
+                    "properties": {
+                        "vm_state": "running",
+                        "vm_name": "sle-micro-55"
```
Comment (mcalmer) [inline scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml:184]:
```suggestion
                        "vm_name": "vm2"
```

**Review by mcalmer — CHANGES_REQUESTED** [review]
example data should not have any reference to real systems in SUSE

**Review by tbazant — COMMENTED** [review]
suggestions applied

File: scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml [hunk 5]
```diff
@@ -332,9 +371,8 @@ backends:
       <title>Customizing &systemd; unit files</title>
       <para>
         To modify the default behavior of the collector's &systemd; unit files,
-        create &systemd; unit <emphasis>drop-in</emphasis> files to override
-        the default behavior, using the <command>systemctl edit</command>
-        mechanism.
+        create &systemd; unit <emphasis>drop-in</emphasis> files to override the
+        default behavior, using the <command>systemctl edit</command> mechanism.
```
Comment (dariavladykina) [inline scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml:375]:
```suggestion
        default behavior using the <command>systemctl edit</command> mechanism.
```

**Review by dariavladykina — COMMENTED** [review]
LGTM, just a minor nit. Thanks!

**Pull request merged**